packages/mutation-testing-elements/src/components/mutation-test-report-popup/mutation-test-report-popup.scss

```diff
@@ -18,18 +18,20 @@
 }
 
 /* Popup arrow */
-.popover::before {
+.popover-arrow {
```

Comment:
I rather use a `::before` pseudo-element instead of an actual div with classes

Reply:
The issue I ran into was that to dynamically set the colour based on the context, the `border-danger` (or other) from bootstrap is used. That class is set from ts code, so the element needs to be accessible from the ts code. For `::before` I couldn't find a way to do that

Reply:
I did find a way, see new implementation 👍

Reply:
Very nice! Does that mean the borders import from bootstrap can also be removed?

Reply:
Yep, we just did 👍